.commitlintrc.ts

```diff
@@ -0,0 +1,52 @@
+import { UserConfig } from '@commitlint/types';
+import { LOCALE_IDS } from './lib/constants/locales';
+import { CALENDAR_IDS } from './lib/constants/calendars';
+
+const empty = [null];
+const calendarScopes = [...CALENDAR_IDS, ...LOCALE_IDS, 'l10n', 'calendar'];
+
+const typesEnumScoped = {
+  // Updates to the automation or release process
+  ci: empty,
+  // Changes to build scripts
+  build: empty,
+  // Documentation only changes
+  docs: empty,
+  // Features, net-new functionality
```

Comment:
Why I assume `net-new` means `totally new`, IMO `new` is sufficient.

```suggestion
  // Features, new functionality
```